src/systemTest/java/com/flightstats/hub/system/resiliency/PlaceholderTest.java

```diff
@@ -0,0 +1,41 @@
+package com.flightstats.hub.system.resiliency;
+
+import com.flightstats.hub.kubernetes.HubLifecycle;
+import com.flightstats.hub.system.extension.SingletonTestInjectionExtension;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.inject.Inject;
+
+/*
+    A temporary example of how SingletonTestInjectionExtension can be used with custom hub setup
+*/
+@Slf4j
+@ExtendWith(SingletonTestInjectionExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class PlaceholderTest {
+    private static final String CUSTOM_YAML_FOR_HELM = "hub: \n" +
```

Comment:
I considered loading an actual yaml file here, it might make these tests a little cleaner, thoughts?

Reply:
It would be nice to maintain these test properties in one place. Setting test properties  manually in multiple places was one of the issues we had in java integration tests resulting in the tight coupling of the tests.